Makefile

```diff
@@ -25,4 +25,4 @@ update: # when requirements.txt changed
 	bash -c "source .env/bin/activate && pip install -r requirements.txt"
 
 test: # run test suite
-	bash -c "source .env/bin/activate && ./manage.py test -v2"
+	bash -c "source .env/bin/activate && coverage run --source='umklapp,umklapp_sitte' ./manage.py test -v2 && coverage report"
```

Comment:
On 02.06.2016 16:54, Andreas Zwinkau wrote:
> 
> You give repo access to people who make such typos? 😉

You're right, maybe I should not do it. But it is soo convenient...

-- 
Karlsruher Institut für Technologie (KIT) IPD Snelting

Martin Mohr
wissenschaftlicher Mitarbeiter

Am Fasanengarten 5, Gebäude 50.34, Raum 022
76131 Karlsruhe

Telefon: +49 721 608-48352
E-Mail: martin.mohr@kit.edu

http://pp.ipd.kit.edu

KIT - Universität des Landes Baden-Württemberg und
nationales Großforschungszentrum in der Helmholtz-Gemeinschaft